borg/archiver.py

```diff
@@ -537,7 +551,7 @@ def do_delete(self, args, repository):
             if not args.cache_only:
                 msg = []
                 try:
-                    manifest, key = Manifest.load(repository)
+                    manifest, key = Manifest.load(repository, Manifest.NO_OPERATION_CHECK)
```

Comment:
minor;

given that every(?) time Manifest.load is called explicitly (outside with_repository) NO_OPERATION_CHECK is passed, make it the default?